app/helpers/application_helper.rb

```diff
@@ -77,4 +77,16 @@ def time_period_class(item)
     return 'currently-valid' if item.currently_valid?
     return 'expired' if item.expired?
   end
+
```

Comment:
I see, but shall we use only small icons for index action ? like current coursemology does. What's your suggestion  ?